docker/entrypoint.sh

```diff
@@ -102,8 +104,30 @@ case ${entry_cmd} in
 				--bind ${CONTAINER_HOST}:${CONTAINER_PORT} \
 				pygeoapi.flask_app:APP
 	  ;;
+
+	# Run pygeoapi server with hot reload
+	hot-reload)
+		# Lock all Python files (for gunicorn hot reload)
+		find . -type f -name "*.py" | xargs chmod 0444
+
+		# SCRIPT_NAME should not have value '/'
+		[[ "${SCRIPT_NAME}" = '/' ]] && export SCRIPT_NAME="" && echo "make SCRIPT_NAME empty from /"
+
+		echo "Start gunicorn name=${CONTAINER_NAME} on ${CONTAINER_HOST}:${CONTAINER_PORT} with ${WSGI_WORKERS} workers and SCRIPT_NAME=${SCRIPT_NAME}"
+		exec gunicorn --workers ${WSGI_WORKERS} \
+				--worker-class=${WSGI_WORKER_CLASS} \
+				--timeout ${WSGI_WORKER_TIMEOUT} \
+				--name=${CONTAINER_NAME} \
+				--bind ${CONTAINER_HOST}:${CONTAINER_PORT} \
+				--reload \
+				--reload-extra-file ${PYGEOAPI_CONFIG} \
+				pygeoapi.flask_app:APP
+			
+		touch ${PYGEOAPI_CONFIG}
```

Comment:
Are you sure this `touch` is actually executed?
As far as I understand it, `exec gunicorn` replaces the bash process with gunicorn and thus stops any further execution of the script, or is this somehow different here?